### PR TITLE
Improve onboarding license creation

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Moves new license creation to the mounted hook of the name team page, but checks if a license exists first. Routing is still predicated on the tenant setting but the overall experience should be smoother. 

Also adds a skip button to bypass the team update sections.

### CODE REVIEW

File: src/pages/Onboard/NameTeam.vue [hunk 1]
```diff
@@ -177,9 +208,19 @@ export default {
       if (this.slugErrors.length > 0) {
         return
       }
-      await this.createLicense()
```
Comment (whimsicallyson) [inline src/pages/Onboard/NameTeam.vue:180]: how do we not need this anymore?

Comment (whimsicallyson) [reply src/pages/Onboard/NameTeam.vue:180]: oh nevermind I missed it above.

**Comment by dylanbhughes** [comment]
@znicholasbrown this looks awesome 💯

**Review by whimsicallyson — APPROVED** [review]
👍  (some larger questions as we discussed in slack, but this is totally good for what we want at the moment)

**Pull request merged**